Update http-events-api.md (#9762)
cloudflare/cloudflare-docs: content/zaraz/http-events-api.md

| `system.page.referrer` | String | The URL of the referrer page in the time the event took place.  |
| `system.page.encoding` | String | The encoding of the page associated with the event. |

{{<Aside type="note">}}

> It's currently not possible to override location related properties, such as City, Country, Continent and alike.

Reviewer: ```suggestion
It is currently not possible to override location related properties, such as City, Country, and Continent.
```
